### DISCUSSION TITLE
Software caused connection abort

### DISCUSSION THREAD

**Discussion by hayvan96** [bitfireAT/icsx5#354] [Q&A] [answered]
Hi, I'm reporting a bug here as indicated in the issue section which seems restricted to vulnerabilities.

Ivve subscribed to an  openagenda which can be downloaded here https://openagenda.com/agendas/80515007/events.v2.ics?relative%5B0%5D=passed&relative%5B1%5D=current&relative%5B2%5D=upcoming

It's often updated and when I open the link on my device in any browser, it successfully downloads a 19 MB ics file.

But since May or so, icsx5 keeps downloading forever and after a long while, it ends with the "software caused connection abort"

How can I keep updating this calendar once a week?
Thanks

**Comment by ArnyminerZ** [comment]
Which Android Version and Device are you using? Have you disabled battery optimizations? Seems to be a system restriction

**Reply by hayvan96** [reply]
Android 14 (LineageOS 21), pixel 6a
No battery optimisation for icsx5.

Have you tried to add the calendar I've mentioned?

**Reply by ArnyminerZ** [reply, marked as answer]
Hi, the link is redirecting to: https://openagenda.com/fr/agendas/80515007/events.v2.ics?relative%5B0%5D=passed&relative%5B1%5D=current&relative%5B2%5D=upcoming

Try to use that one.

Please, take into account that this is a 18.8MB file. It's quite large for what icals usually are, so it will take a while for sure. It's even probable that your device will cut the transmission because of this. For me it's working fine, but it's taking almost 3 minutes to fetch it.

**Reply by hayvan96** [reply]
I've found another link which on list current and future events, not passed, and the file is a lot smaller. Seems to work fine (it's instant).
